tests/test_get_public_key.py

```diff
@@ -3,26 +3,25 @@
 from inspect import currentframe
 from ragger.navigator import NavInsID, NavIns
 from bip_utils import Bip39SeedGenerator, Bip32Secp256k1
-from time import sleep
-
+from time import sleep, time
+from conftest import wait_for_home_screen
 DEFAULT_SEED = "glory promote mansion idle axis finger extra february uncover one trip resource lawn turtle enact monster seven myth punch hobby comfort wild raise skin"
 
 def test_get_public_key(cmd,navigator,firmware,backend):
-    if firmware.device == "stax":
-        sleep(4)
-    else:
-        sleep(1)
+    wait_for_home_screen(backend,firmware)
+    
     result: list = []
 
     bip32_path="44'/60'/0'/0/0"
 
     path = Path(currentframe().f_code.co_name)
     
     with cmd.get_public_key(bip32_path=bip32_path, result=result) as ex:
-        sleep(1)
         if firmware.device.startswith("nano"):
+            sleep(1)
             navigator.navigate_and_compare(Path(__file__).parent.resolve(),path,[NavIns(NavInsID.BOTH_CLICK),NavIns(NavInsID.BOTH_CLICK)],screen_change_after_last_instruction = False)
         else:
+            sleep(3)
```

Comment:
Longer sleep becaue Stax is slower ? Super platform dependent and will break when Speculos perfs change

Reply:
This is ugly indeed. I might have to add another "wait_for" function...

Reply:
New `wait_for_text` function has been added.